src/Core/Test/Stub/Framework/IdsCollection.php

```diff
@@ -142,7 +142,7 @@ public function getKeys(array $ids): string
             if ($key) {
                 $keys[] = $key;
             } else {
-                throw new \RuntimeException('Key not found for id ' . $id);
+                $keys[] = 'Key not found for ID ' . $id;
```

Comment:
ehm was this for debugging purposes or intentional? Looks like it only hides problems?

Reply:
https://github.com/shopware/shopware/pull/6607#discussion_r1943002949